client/components/MessageList.vue

```diff
@@ -63,6 +63,8 @@ import Message from "./Message.vue";
 import MessageCondensed from "./MessageCondensed.vue";
 import DateMarker from "./DateMarker.vue";
 
+let unreadMarkerShown = false;
```

Comment:
For context, this variable wasn't defined as `data`, and if you do that it will become reactive and cause updates in an infinite loop.